packages/ses/NEWS.md

```diff
@@ -21,7 +21,11 @@ User-visible changes in SES:
   [Tracking issue for getting 3rd party packages more SES friendly (#576)](https://github.com/endojs/endo/issues/576)
   we track the incompatibilities we encounter and progress toward
   getting them fixed.
-
+- Add utility function `Compartment.prototype.__isKnownScopeProxy__(value)` to
+  aid working around scopeProxy leakage. Returns true if `value` is one of the
```

Comment:
```suggestion
  aid working around [scopeProxy leakage](https://github.com/endojs/endo/issues/31). Returns true if `value` is one of the
```

If there's also a good place to link to https://github.com/endojs/endo/issues/622 , please do so as well.

Reply:
I should qualify "reviewed the whole thing.". I did not review packages/ses/test/test-compartment-known-scope-proxy.js closely. I am ok proceeding without doing so if you are, especially given @kriskowal 's contingent thumbs up.